src/StreamTags.php

```diff
@@ -0,0 +1,109 @@
+<?php
```

Comment:
If you agree that a `StreamTag` is a `Segment`, this class is no longer necessary.